weasyl/test/web/test_interaction.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from weasyl import errorcode
+from weasyl.test import db_utils
+
+
+INVALID_PATHS = (
+    '/note',
+    '/note?',
+    '/note?noteid',
+    '/note?noteid=',
+    '/note?noteid=a',
```

Comment:
These tests are also overkill for something that’s more about integer parsing than notes.

Reply:
OK if I move these to a separate `test_forms.py` that just tests `expect_id` directly, or would you want most of these gone entirely?